.github/workflows/schema-validation.yml

```
@@ -46,56 +52,82 @@ jobs:
+            --artifacts-dir artifacts/schema_authority \
+            --diff-out artifacts/schema_authority/canonical_drift.diff
+
+      - name: Runtime identity parity check
+        env:
+          DATABASE_URL: postgresql://app_user:app_user@localhost:5432/skeldir_test
+          MIGRATION_DATABASE_URL: postgresql://migration_owner:migration_owner@localhost:5432/skeldir_test
+          EXPECTED_RUNTIME_DB_USER: app_user
+        run: |
+          python scripts/identity/assert_runtime_identity.py
+
+      - name: Negative control: runtime identity mismatch must fail
```

actionlint (1.7.10)

[error] could not parse as YAML: mapping values are not allowed in this context

(syntax-check)